CGPs/cgp-0046.md

```diff
@@ -0,0 +1,104 @@
+---
+cgp: 46
+title: cREAL oracle and constitution parameter activation
+date-created: 2021-11-01
+author: @martinvol
+status: DRAFT
+discussions-to: [cBRL forum proposal](https://forum.celo.org/t/cbrl-stable-asset/1281)
+governance-proposal-id: [if submitted]
+date-executed: [if executed] <date created on, in ISO 8601 (yyyy-mm-dd) format>
+---
+## Overview
+
+CGP 46 - cREAL oracle, freezing & constitution parameter activation
+
+
+### Status
+
+This proposal is meant to be executed on-chain before after [CGP-42](https://github.com/celo-org/governance/blob/main/CGPs/cgp-0042.md) and [CGP-45](https://github.com/celo-org/governance/blob/main/CGPs/cgp-0045.md). It enables the oracles designated to cREAL and sends 100 CELO to each of them to pay for gas for a couple of years.
+
+It also does some administrative miscellaneous, namely
+
+It also freezes the contracts StableTokenBRL and ExchangeBRL at the time of launch, it can 
```

Comment:
this also feels like an incomplete sentence. What was the intention after "it can".

Furthermore, are the contract names finalized or can it be modified to StableTokenREAL and ExchangeREAL?

Reply:
The name of the oracle will remain as they are, as the currency they track is BRLL.